src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,45 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int totalSupply = 0;
+        int totalBuy = 0;
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName));
+             BufferedWriter writer = new BufferedWriter(new FileWriter(toFileName))) {
+
+            String line;
+            while ((line = reader.readLine()) != null) {
+                String[] parts = line.split(",");
+                String operationType = parts[0].trim();
+                int amount = Integer.parseInt(parts[1].trim());
+
+                if ("supply".equals(operationType)) {
```

Comment:
The string literals 'supply' and 'buy' are used multiple times and should be declared as constants to avoid repetition and to make the code more maintainable.

Reply:
not fixed